codec-http2/src/test/java/io/netty/handler/codec/http2/Http2SettingsTest.java

```diff
@@ -86,4 +87,21 @@ public void boundarySettingsShouldBeSet() {
         settings.maxHeaderListSize((int) settingsValueUpperBound);
         assertEquals(Integer.MAX_VALUE, (long) settings.maxHeaderListSize());
     }
+
+    @Test
+    public void headerTableSizeUnsignedInt() {
+        final long value = 1L << 31;
```

Comment:
should we instead check around the bounds `MAX_HEADER_TABLE_SIZE` and `MAX_HEADER_TABLE_SIZE+1`?